build/cache/cache.go

```diff
@@ -77,11 +77,6 @@ type BuildCache struct {
 	GOPATH    string
 	BuildTags []string
 	Minify    bool
-	// When building for tests, import path of the package being tested. The
-	// package under test is built with *_test.go sources included, and since it
-	// may be imported by other packages in the binary we can't reuse the "normal"
-	// cache.
-	TestedPackage string
```

Comment:
One thing that's not clear to me is if we should share a cache between `gopherjs test` and `gopherjs build`.

For example, let's say `foo` depends on `shared`, and you run `gopherjs test foo/...`. This will create a cache for `fake_dep/shared`. If you run a subsequent `gopherjs build`, is it okay to reuse the cache for `fake_dep/shared`?

I expect this would be fine as long as the rest of the `BuildCache` args are the same.